src/Neo/UInt160.cs

```diff
@@ -101,6 +101,16 @@ public override int GetHashCode()
             return HashCode.Combine(_value1, _value2, _value3);
         }
 
+        /// <summary>
+        /// Gets a ReadOnlySpan that represents the current value in little-endian.
+        /// </summary>
+        /// <returns>A ReadOnlySpan that represents the current value in little-endian.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public ReadOnlySpan<byte> GetSpan()
+        {
+            return MemoryMarshal.CreateReadOnlySpan(ref Unsafe.As<ulong, byte>(ref _value1), Length);
```

Comment:
I think this will be platform dependant, in a BigEndian machine we will get different results

Reply:
> I think this will be platform dependant, in a BigEndian machine we will get different results

The `StructLayout` of `UInt160` is little endian, so if run in a in a BigEndian machine, `UInt160` should be adapted.
But I think is not a problem, the machines that can be bought and can run `neo` are all  little endian. It is enough to support only little-endian.

Reply:
We must fix it, it must be deterministic

Reply:
> We must fix it, it must be deterministic

fixed

Reply:
> We must fix it, it must be deterministic

But I think it not necessary.
`UInt160.Serialize` uses `BinaryWriter`, and `BinaryWriter.Write` writes `int/long` in little endian, so it is little endian always.
https://github.com/dotnet/runtime/blob/0170d78578a04206542f32e4db3eaa88a5384973/src/libraries/System.Private.CoreLib/src/System/IO/BinaryWriter.cs#L314